lib/Targeting/Storage/SessionStorage.php

```diff
@@ -184,6 +184,10 @@ private function getSessionBag(VisitorInfo $visitorInfo, string $scope, bool $ch
                 ));
         }
 
+        if (!$bag instanceof NamespacedAttributeBag) {
```

Comment:
not possible anymore to return null....so please change phpdoc & all usages e.g.
https://github.com/pimcore/pimcore/blob/0bbb16077609757bb698dad1593c65eabb4088fe/lib/Targeting/Storage/SessionStorage.php#L127